### PR TITLE
Include render message on activate

### PR DESCRIPTION
**Pull request by luisalejandrosalazar**
The render message functionality on activate role has been added.

### CODE REVIEW

File: src/pages/privileges/outlets/roles/add-role/index.tsx [hunk 1]
```diff
@@ -7,8 +7,8 @@ import { dataToAssignmentFormEntry } from "@pages/privileges/outlets/linixUseCas
 import { IFormAddRole, IFormAddRoleRef, IInitialiceFormRole } from "../types";
 import { addRoleStepsRules } from "./utils";
 import { stepsAddRol } from "./config/addRol.config";
-import { IGeneralInformationForm } from "./forms/GeneralInformationForm";
-import { IAncillaryAccountsForm } from "./forms/AncillaryAccounts";
+import { IGeneralInformationForm } from "../components/GeneralInformationForm";
+import { IAncillaryAccountsForm } from "../components/AncillaryAccountsForm";
 import { AddRolUI } from "./interface";
 import { initialValuesAddRol } from "./config/initialValues";
```
Comment (LinconOssa) [inline src/pages/privileges/outlets/roles/add-role/index.tsx:14]: image
This needs to be created so that there are no errors.

**Review by LinconOssa — COMMENTED** [review]
image
1The problem needs to be resolved here in hasChanges and handleSubmit

2.for FormButtons fix like this
image

File: src/pages/privileges/outlets/roles/add-role/index.tsx (hunk 1 above)
Comment (luisalejandrosalazar) [reply src/pages/privileges/outlets/roles/add-role/index.tsx:14]: Types have been adjusted.

**Review by LinconOssa — APPROVED** [review]
Meets the requirements.

File: src/components/forms/submit/FormButtons/index.tsx [hunk 2]
```diff
@@ -2,8 +2,8 @@ import { Button, Stack, inube } from "@inube/design-system";
 import { ThemeProvider } from "styled-components";
 
 interface FormButtonsProps {
-  children: React.ReactNode;
-  handleSubmit: () => void;
+  children?: React.ReactNode;
```
Comment (carin91) [inline src/components/forms/submit/FormButtons/index.tsx:5]: optional props must be placed at the end of the list of props.

File: src/components/forms/submit/FormButtons/index.tsx [hunk 3]
```diff
@@ -2,8 +2,8 @@ import { Button, Stack, inube } from "@inube/design-system";
...
+  handleSubmit: (name: string, value: string) => void;
   handleReset: () => void;
   disableReset?: boolean;
```
Comment (carin91) [inline src/components/forms/submit/FormButtons/index.tsx:8]: adjust the props list based on alphabetical order, first the mandatory props in alphabetical order, latter the optional props in alphabetical order.

**Review by carin91 — APPROVED** [review]

**Pull request merged**